src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -240,7 +240,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             debugPlusMode As Boolean,
             xmlReferenceResolver As XmlReferenceResolver,
             sourceReferenceResolver As SourceReferenceResolver,
-            SyntaxTreeOptionsProvider As SyntaxTreeOptionsProvider,
```

Comment:
Parameter usages are already in lowercase. But the IDE was automatically turning them PascalCased on save because of the actual parameter name here.